src/tools/ts-generator/linksTSgenerator.cpp

```diff
@@ -0,0 +1,346 @@
+
+#include "linksTSgenerator.h"
+#include <yuni/datetime/timestamp.h>
+
+namespace Antares::TSGenerator
+{
+// ==================
+// Free functions
+// ==================
+std::vector<std::string> extractTargetAreas(fs::path sourceLinkDir)
+{
+    std::vector<std::string> to_return;
+    fs::path pathToIni = sourceLinkDir / "properties.ini";
+    IniFile ini;
+    ini.open(pathToIni); // gp : we should handle reading issues
+    for (auto* s = ini.firstSection; s; s = s->next)
+    {
+        std::string targetAreaName = transformNameIntoID(s->name);
+        to_return.push_back(targetAreaName);
```

Comment:
```suggestion
        to_return.push_back(transformNameIntoID(s->name));
```

Reply:
I did this, because it makes code shorter, but intermediate variable **targetAreaName** had the advantage to be very clear.

Reply:
Done